addons/website_event_exhibitor/static/src/js/event_exhibitor_connect.js

```diff
@@ -22,11 +23,11 @@ publicWidget.registry.eventExhibitorConnect = publicWidget.Widget.extend({
     start: function () {
         var self = this;
         return this._super(...arguments).then(function () {
-            self.eventIsOngoing = self.$el.data('eventIsOngoing') || false;
-            self.sponsorIsOngoing = self.$el.data('sponsorIsOngoing') || false;
-            self.isParticipating = self.$el.data('isParticipating') || false;
-            self.userEventManager = self.$el.data('userEventManager') || false;
-            self.$el.on('click', self._onConnectClick.bind(self));
+            self.eventIsOngoing = self.el.dataset.eventIsOngoing || false;
+            self.sponsorIsOngoing = self.el.dataset.sponsorIsOngoing || false;
+            self.isParticipating = self.el.dataset.isParticipating || false;
+            self.userEventManager = self.el.dataset.userEventManager || false;
+            self.el.addEventListener("click", self._onConnectClick.bind(self));
```

Comment:
I know it was like that before, but shouldn't the event listener be removed in a `destroy` method ?

Reply:
@bso-odoo Yeah it should have been removed, but this code never had destroy method hence we didnt introduced anything new to the code, but yeah for the sake of removing listener i will add one to it.
Thanks!